build-common/common.xml

```diff
@@ -11,15 +11,19 @@
 	<!-- Path to the folder that contain the external assemblies -->
 	<property name="lib.dir" value="lib" dynamic="true" />
 
-	<!-- This is used only for build folder -->
-	<!-- TODO: Either remove or refactor to use NHibernate.props -->
-	<property name="project.version" value="5.2.6" overwrite="false" />
-	<property name="project.version.numeric" value="5.2.6" overwrite="false" />
```

Comment:
`project.version.numeric` is used to build the documentation.

Reply:
What's use of it? I see it's used [here](https://github.com/nhibernate/nhibernate-core/blob/ac39173567b31bcfad475ca32687c9faf0d37f87/doc/NHibernate.shfbproj.template#L26). But I don't see any representation of it in generated help files on master. And current help files also seems fine (except 5.2 version is stated in docs but it's hardcoded in xml).

Reply:
I nuked it - everything seems OK. Are you sure we need it?

Reply:
OK I see it's about "NHibernate API" docs which is not part of any build. But still I see no point in additional property - let's use `project.version` there too.